Userland/Libraries/LibGfx/SystemTheme.cpp

```diff
@@ -40,8 +40,23 @@ Core::AnonymousBuffer load_system_theme(Core::ConfigFile const& file)
     auto get_color = [&](auto& name) {
         auto color_string = file.read_entry("Colors", name);
         auto color = Color::from_string(color_string);
-        if (!color.has_value())
-            return Color(Color::Black);
+        if (!color.has_value()) {
+            auto maybe_color_config = Core::ConfigFile::open(data->path[(int)PathRole::ColorScheme]);
+            if (maybe_color_config.is_error())
+                maybe_color_config = Core::ConfigFile::open("/res/color-schemes/Default.ini");
+            auto color_config = maybe_color_config.release_value();
+            if (strcmp(name, "ColorSchemeBackground") == 0)
```

Comment:
Why use strcmp here? Can't we just use `==` against a stringview?
`if (name == "ColorSchemeBackground"sv)`